bibtex-actions-file.el

```diff
@@ -177,11 +180,10 @@ If you use 'org-roam' and 'org-roam-bibtex', you can use
 
 (defun bibtex-actions-file-open-notes-default-org (key entry)
   "Open a note file from KEY and ENTRY."
-  ;; modify when this addressed:
-  ;; https://github.com/org-roam/org-roam-bibtex/issues/211
   (if-let* ((file
              (caar (bibtex-actions-file--files-to-open-or-create
-                    (list key)
+                    key
+                    entry
                     bibtex-actions-notes-paths '("org"))))
             (file-exists (file-exists-p file)))
```

Comment:
`files-to-open-or-create` already checks whether file exists or not and that is in the cons cell so this is redundant. But this is also the only place `files-to-open-or-create`is used as far as I can tell. So maybe that can be removed and instead inlined here.

Reply:
That's what i was thinking, and plan to do.